pkg/cmd/pr/checkout/checkout.go

```diff
@@ -96,7 +96,9 @@ func checkoutRun(opts *CheckoutOptions) error {
 	}
 
 	headRemote := baseRemote
-	if pr.IsCrossRepository {
+	if pr.HeadRepository == nil {
+		headRemote = nil
+	} else if pr.IsCrossRepository {
```

Comment:
This is just a short circuit, correct? `remotes.FindByRepo` was previously returning nil when `pr.HeadRepository == nil`?

Reply:
Correct! But now `pr.HeadRepository.Name` will crash when there is no head repository, so we have to guard against accessing it.